packages/map-template/src/components/Map/GoogleMapsMap/GoogleMapsMap.jsx

```diff
@@ -39,7 +41,8 @@ function GoogleMapsMap({ onMapView, onPositionControl }) {
                 // Always set a center so the map so the bounds or center can be read from the start.
                 center: { lat: 0, lng: 0 },
                 // Set a large zoom so we prevent a "zoom 0 glitch" (showing the whole globe temporarily)
-                zoom: 21
+                zoom: 21, 
+                mapId: gmMapId
```

Comment:
Would it be more safe to exclude completely if not set? Does it work as expected when passing the default `null`?

Reply:
The atom has `null` as default. If that works, then no need to change anything 😄

Reply:
Yeah, it works 👍